src/main/java/me/coley/recaf/search/SearchCollector.java

```diff
@@ -62,7 +65,8 @@ public SearchCollector(Workspace workspace, Collection<Query> queries) {
 	public List<SearchResult> getOverlappingResults() {
 		// Get results of multiple queries that have either the same parent context
 		// (or just same context in some cases)
-		return results.values().stream()
+		// TODO: remove asMap() call?
+		return (List<SearchResult>) results.asMap().values().stream()
```

Comment:
Casting like this is incorrect:

See results of `mvn test` _(`mvn test -Dcheckstyle.skip=true` in this case, see comments on `ClasspathUtil`)_

![image](https://user-images.githubusercontent.com/21371686/62822887-c6996000-bb57-11e9-8f48-557bfe00b08a.png)